src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -215,28 +194,56 @@ export class MeetingManager implements AudioVideoObserver {
   }
 
   async join({
-    meetingInfo,
-    attendeeInfo,
-    deviceLabels = DeviceLabels.AudioAndVideo,
+    meetingSessionConfiguration,
+    deviceLabels,
     eventReporter,
     meetingManagerConfig,
-  }: MeetingJoinData) {
+  }: MeetingJoinData): Promise<void> {
+    if (deviceLabels === null || deviceLabels === undefined) {
```

Comment:
Sorry, if I have missed this in last review. I had a comment, should we default to `DeviceLabels.AudioAndVideo`? Looks like it can be a default behavior and if builders want can override?

Reply:
Yeah we should not change that :)